tests/samplers_tests/test_samplers.py

```diff
@@ -223,15 +166,13 @@ def test_discrete_uniform(
         np.floating,
     )
 
-    # Check all points are multiples of distribution.q.
-    points = points
-    points -= distribution.low
-    if isinstance(distribution, DiscreteUniformDistribution):
-        points /= distribution.q
-    else:
+    if distribution.step is not None:
+        # Check all points are multiples of distribution.step.
+        points = points
```

Comment:
I just found `points = points`. 💭 I think we can remove it.
@toshihikoyanase Do you have any specific reason to leave it? The change was introduced in:

https://github.com/optuna/optuna/commit/184e867d1a58082d397600da21a2c6d4f6fa4aaa#diff-8b85ccf8a3534b2aa878bce8ece8ce930af410ab178ebc6d80031badbabb5052L108-R73

Reply:
I asked @toshihikoyanase and confirmed it leaves unintentionally. Please remove it as well. :pray:

Reply:
Done in 2b8034e9d4710ea9fea3661371d295fce963d9ea!